sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -174,8 +174,6 @@ private[hive] class IsolatedClientLoader(
 
   // Pre-reflective instantiation setup.
   logDebug("Initializing the logger to avoid disaster...")
-  Thread.currentThread.setContextClassLoader(classLoader)
```

Comment:
This probably causes ClassNotFoundException or ClassCastException if we removed this line, as we do need to load the specified version of Hive Metastore jars and its dependencies.